api/.env.example

```diff
@@ -0,0 +1,74 @@
+##########################################
+# Environment Variables for CodePush Server
+##########################################
+
+# ==============================
+# Storage Configuration (REQUIRED - choose one)
+# ==============================
+EMULATED=false # Set to 'true' to use the local emulator
+
+# --- Azure Storage Configuration ---
+AZURE_STORAGE_ACCOUNT=    # Azure storage account name
+AZURE_STORAGE_ACCESS_KEY= # Azure storage access key (if KeyVault not used)
+
+# ==============================
+# Server Configuration (REQUIRED)
+# ==============================
+SERVER_URL=http://localhost:3000 # The URL of your server
+
+# ==============================
+# Authentication (REQUIRED - at least one provider)
+# ==============================
+
+# --- GitHub OAuth ---
+GITHUB_CLIENT_ID=     # GitHub OAuth client ID
+GITHUB_CLIENT_SECRET= # GitHub OAuth client secret
+
+# --- Microsoft OAuth ---
+MICROSOFT_CLIENT_ID=     # Microsoft OAuth client ID
+MICROSOFT_CLIENT_SECRET= # Microsoft OAuth client secret
+
+# ==============================
+# Optional Configuration
+# ==============================
+
+# --- HTTPS Configuration ---
+HTTPS= # Set to 'true' to enable HTTPS for local deployment
+
+# --- Debugging Configuration ---
+LOGGING=false            # Enable CodePush-specific logging
+DEBUG_DISABLE_AUTH=false # Disable OAuth authentication route
+DEBUG_USER_ID=           # Backend user ID for debugging session
+
+# ==============================
+# Redis Configuration
+# ==============================
+REDIS_HOST=     # Redis server IP address
+REDIS_PORT=6379 # Redis port (default: 6379)
+REDIS_KEY=      # Redis authentication key
+
+# ==============================
+# Unit Testing Configuration
+# ==============================
+TEST_AZURE_STORAGE=false  # Run API unit tests against Azure storage
+AZURE_STORAGE_ACCOUNT=    # Storage account for testing (if TEST_AZURE_STORAGE=true)
+AZURE_STORAGE_ACCESS_KEY= # Storage access key for testing
```

Comment:
AZURE_STORAGE_ACCOUNT=    # Storage account for testing (if TEST_AZURE_STORAGE=true)
    AZURE_STORAGE_ACCESS_KEY= # Storage access key for testing

This is duplication of the same settings above which will override them. Every setting should be only once in the example.

Reply:
You're right, I just removed them.